content/en/docs/contribute/generate-ref-docs/kubernetes-components.md

```diff
@@ -91,7 +73,43 @@ When the Markdown files are in your local clone of the `kubernetes/website`
 repository, you can submit them in a [pull request](/docs/contribute/start/)
 to `kubernetes/website`.
 
-## Customizing the config file
+## Configuration file format
+
+Each config file may contain multiple repos that will be imported together.
+When necessary, you can customize the configuration file by manually editing
+it. You may create new config files for importing other groups of documents. Imported documents must follow these guidelines:
+
+1. Adhere to the [Documentation Style Guide](/docs/contribute/style/style-guide/).
+
+1. Have `title` defined in the front matter. For example:
+
+    ```
+    ---
+    title: Title Displayed in Table of Contents
+    ---
+
+    Rest of the .md file...
+    ```
+1. Be listed in the `kubernetes/website/data/reference.yml` file
+
+The following is an example of the YAML configuration file:
+
+```yaml
+repos:
+- name: community
+  remote: https://github.com/kubernetes/community.git
+  branch: master
+  files:
+  - src: contributors/devel/README.md
+    dst: docs/imported/community/devel.md
+  - src: contributors/guide/README.md
+    dst: docs/imported/community/guide.md
+```
+
+Note: `generate-command` is an optional entry, which can be used to run a
+given command or a short script to generate the docs from within a repo.
```

Comment:
Do we need to include this or should it be removed since we tell people to use reference.yml?

Reply:
I'm more inclined to leave it there.